src/main/java/com/gluonhq/substrate/target/IosTargetConfiguration.java

```diff
@@ -41,33 +41,80 @@
+            "com.sun.glass.ui.ios.IosPixels",
+            "com.sun.glass.ui.ios.IosView",
+            "com.sun.glass.ui.ios.IosWindow"
+    );
+
+    private static final List<String>javafxJNIIosClassList = Arrays.asList(
+            "com.sun.glass.ui.ios.IosApplication",
+            "com.sun.glass.ui.ios.IosCursor",
+            "com.sun.glass.ui.ios.IosGestureSupport",
+            "com.sun.glass.ui.ios.IosMenuBarDelegate",
+            "com.sun.glass.ui.ios.IosPixels",
+            "com.sun.glass.ui.ios.IosView",
+            "com.sun.glass.ui.ios.IosWindow",
+            "java.util.Vector",
+            "com.sun.javafx.font.coretext.CGAffineTransform",
+            "com.sun.javafx.font.coretext.CGPoint",
+            "com.sun.javafx.font.coretext.CGRect",
+            "com.sun.javafx.font.coretext.CGSize",
+            "com.sun.javafx.font.FontConfigManager$FcCompFont",
+            "com.sun.javafx.font.FontConfigManager$FontConfigFont",
+            "com.sun.javafx.iio.ios.IosImageLoader"
+    );
+
+    private static final List<String> ioslibs = Arrays.asList(
+            "-lpthread", "-lz", "-lstrictmath", "-llibchelper",
+            "-ljava", "-ljvm", "-lnio", "-lzip", "-lnet");
+
+    private static final List<String> javafxLibs = Arrays.asList(
+            "prism_es2", "glass", "javafx_font", "prism_common", "javafx_iio");
 
     @Override
     List<String> getTargetSpecificLinkFlags(boolean useJavaFX, boolean usePrismSW) {
-        return Arrays.asList("-w", "-fPIC",
+        List<String> linkFlags = new ArrayList<>(Arrays.asList("-w", "-fPIC",
                 "-arch", Constants.ARCH_ARM64,
                 "-mios-version-min=11.0",
-                "-isysroot", getSysroot(),
-                "-Wl,-framework,Foundation",
-                "-Wl,-framework,UIKit",
-                "-Wl,-framework,CoreGraphics",
-                "-Wl,-framework,CoreText",
-                "-Wl,-framework,OpenGLES",
-                "-Wl,-framework,MobileCoreServices");
+                "-isysroot", getSysroot()));
+        if (useJavaFX) {
+            String javafxSDK = projectConfiguration.getJavafxStaticLibsPath().toString();
+            javafxLibs.forEach(name ->
+                    linkFlags.add("-Wl,-force_load," + javafxSDK + "/lib" + name + ".a"));
+        }
+        linkFlags.addAll(ioslibs);
+        linkFlags.addAll(Arrays.asList(
+                "-Wl,-framework,Foundation", "-Wl,-framework,UIKit",
+                "-Wl,-framework,CoreGraphics", "-Wl,-framework,MobileCoreServices",
+                "-Wl,-framework,OpenGLES", "-Wl,-framework,CoreText",
+                "-Wl,-framework,QuartzCore", "-Wl,-framework,ImageIO"));
+        return linkFlags;
     }
 
     @Override
     List<String> getTargetSpecificCCompileFlags() {
         return Arrays.asList("-xobjective-c",
                 "-arch", getArch(),
-                "-Dsvn.targetArch=" + getArch(),
+                "-Dsvm.targetArch=" + getArch(),
```

Comment:
why do we need this here?

Reply:
It was set [here](https://github.com/johanvos/substrate/blob/ioslibs/src/main/java/com/gluonhq/substrate/target/IosTargetConfiguration.java#L61), but it is not needed, indeed. 
I'll remove it.